Woof/Woof/View/OwnerCardView/OwnerCardView.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+/// A view displaying information about the pet owner.
+struct OwnerCardView: View {
+    /// The view model responsible for providing data to the view.
+    @ObservedObject var viewModel: OwnerCardViewModel
+
+    var body: some View {
+        HStack {
+            AvatarView(url: viewModel.imageURL)
+
+            VStack(alignment: .leading, spacing: 8) {
```

Comment:
magic number?

Reply:
Thanks, fixed it.

Reply:
So why you don't create a view model for reusable views if you follow this rule?
For example `LabelWithTextAndImage`

Reply:
You are right, agree. As we discussed, the viewModel are deleted.